app/templates/constrainttemplates.html

```diff
@@ -65,7 +66,7 @@ <h5 class="header">Parameters schema</h5>
         </div>
         
         <div class="ui horizontal very relaxed divided list content">
-          {% for property, value in constrainttemplate.spec.crd.spec.validation.openAPIV3Schema.properties.items() %}
+          {% for property, value in constrainttemplate.spec.crd.spec.validation.openAPIV3Schema.get("properties",{}).items() %}
```

Comment:
Change a little bit the check.
Now the If will check if there is `openAPIV3Schema` at all defined. And here will check if there are any properties, if not will skip, as by default is empty dict.

Reply:
This would make the section `Parameters` appear in the UI but empty when there are no properties. We should either hide it (like we do until now) or show a message saying that there are no properties like we do for the violations. In this particular case, I'm more inclined to just hide the section.

Reply:
Currently it's the same case if there are properties which are empty. Still, agree it will be better to hide it.

What do you think about making check for properties length before creating the section?

This will take care if openAPIV3Schema/validation is missing or properties are missing/empty.